### DISCUSSION TITLE
Security Certificates expiring

### DISCUSSION THREAD

**Discussion by mattdandrews** [Security-Onion-Solutions/securityonion#13294] [General]
I have a number of security certificates that have expired or on some servers are about to expire: Not After : Jul 24 11:00:20 2024 GMT. I use self-signed certificates generated at install. Is there an easy way to re-issue all of the Security Onion generated certificates without having to do this manually? I am running several versions based on Centos 2.3.60+

**Comment by dougburks** [comment]
Please note that Security Onion 2.3 has reached End Of Life and is no longer supported:
https://blog.securityonion.net/2023/10/6-month-eol-notice-for-security-onion-23.html

Please migrate to Security Onion 2.4:
https://blog.securityonion.net/2024/06/security-onion-2480-now-available.html
